scripts/h5peditor-av.js

```diff
@@ -380,14 +380,9 @@ H5PEditor.widgets.video = H5PEditor.widgets.audio = H5PEditor.AV = (function ($)
     
     this.$errors = $container.children('.h5p-errors');
 
-    if (this.params !== undefined) {
```

Comment:
Just mentioning because of the commit message: You can have the `index` in `map` as well. It would be the second argument of the closure.